organizers/2020-02-24.html

```diff
@@ -0,0 +1,145 @@
+---
+layout: organizers
+---
+<!-- Syntax reference: https://github.com/hakimel/reveal.js/blob/master/index.html -->
+
+<section>
+    <h1>The London Django Meetup Group</h1>
+    <h2>February 24, 2020</h2>
+    <img data-src="/img/django-logo-white-transparent.svg" style="width:40%; background: none" alt="Django logo" class="plain">
+</section>
+
+<section>
+    <h1>👋 Welcome All!</h1>
+</section>
+
+<section>
+    <h1>Code of conduct</h1>
+    <h2>😃 Be Awesome to Each Other </h2>
+    <p>The mission of The London Django Meetup Group is to provide a fun, welcoming and professional environment for a diverse group of people.</p>
+    <p>
+        Code of conduct: <a href="https://www.djangolondon.com/code-of-conduct/">djangolondon.com/code-of-conduct</a>
+    <p>
+</section>
+
+<section data-background='#eee'>
+    <h1>Sponsorship</h1>
+    <p>Tonight's meetup is kindly sponsored by:</p>
+    <ul>
+      <li><strong>Festicket</strong></li>
+      <li><strong>Pollen (formerly Verve)</strong></li>
+      <li><strong>Time Out</strong></li>
+    </ul>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/festicket.svg" style="width:40%" alt="Festicket" class="plain">
+    <p><em>
+        Europe's largest music festival platform, and UK's 6th fastest growing private tech start up.<br>Transforming the way music fans discover, book tickets and travel packages for festivals around the world.
+    </em></p>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/pollen.png" style="width:40%" alt="Pollen" class="plain">
+    <p><em>
+        Pollen is a marketplace for the best in shared experiences.
+    </em></p>
+    <p><small>(Formerly Verve)</small></p>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/timeout.png" style="width:40%" alt="Time Out" class="plain">
+    <p><em>
+        The trusted global platform that inspires and enables people to experience the best of the city. Operating in 107 cities, across 39 countries, it has an average monthly global audience reach of over 111 million across all platforms.
+    </em></p>
+</section>
+
+<section>
+    <h1>📋 Today's Schedule</h1>
+    <h2>🐍 News </h2>
+    <h2>🎤 Talk </h2>
+    <h2>🎁 Prize Draw </h2>
+    <h2>🍕 Food &amp; Drink </h2>
+</section>
+
+
+<section>
+    <h1>🐍 News</h1>
+    <h2>Django security release</h2>
+    <p>Potential SQL injection via StringAgg</p>
+    <p>https://www.djangoproject.com/weblog/2020/feb/03/security-releases/</p>
+</section>
+
+
+<section>
+    <h1>🐍 News</h1>
+    <h2>DjangoCon Europe</h2>
+    <p>May 27-31 - Porto, Portugal 🇵🇹</p>
+    <p>CFP Deadline is 2020-02-29</p>
+    <p>https://2020.djangocon.eu/</p>
+</section>
+
+
+<section>
+    <h1>🐍 News</h1>
+    <h2>Django Day</h2>
+    <p>April 17th - Copenhagen, Denmark 🇩🇰</p>
+    <p>https://djangoday.dk/</p>
+</section>
+
```

Comment:
Djangocongress JP has been announced, just in case anyone wants to go Japan: https://djangocongress.jp/

```
<section>
    <h1>🐍 News</h1>
    <h2>DjangoCongress JP</h2>
    <p>June 20 - Nagano, Japan 🇯🇵</p>
    <p>https://djangocongress.jp/</p>
</section>
```

Reply:
👍